src/index.html

```diff
@@ -10,13 +10,21 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+
     <title>Stop watch</title>
+
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="wrapper">
+      <div class="stopwatch stopwatch--speed-up">
+        <div class="stopwatch__minutes"></div>
+        <div class="stopwatch__seconds"></div>
```

Comment:
the same)

Reply:
stopwatch__second-hand looks better, right?))